src/course-unit/course-xblock/CourseXblock.jsx

```diff
@@ -0,0 +1,73 @@
+import PropTypes from 'prop-types';
+import { Dropdown, Icon, IconButton } from '@edx/paragon';
+import { EditOutline as EditIcon, MoreVert as MoveVertIcon } from '@edx/paragon/icons';
+import { useIntl } from '@edx/frontend-platform/i18n';
+
+import messages from './messages';
+
+const CourseXblock = ({ id, title }) => {
+  const intl = useIntl();
+
+  return (
+    <li className="studio-xblock-wrapper mb-4">
+      <section className="wrapper-xblock p-4">
+        <div className="xblock-header d-flex justify-content-between mb-4">
+          <div className="header-details">
+            <span className="h3 xblock-display-name">{title}</span>
```

Comment:
[optional] You are using the title class, but it seems to me that the element should be a title

Reply:
Replaced with the paragon Card component